CONTRIBUTING.md

```diff
@@ -2,21 +2,53 @@
 
 Observable Plot is released under the ISC license. You are welcome to fork this repository and to send us pull requests to contribute bug fixes or new features.
 
-To develop Observable Plot locally, clone this repository and install its dependencies:
+## Set up
+To set up your local development environment for Plot, follow these steps on your terminal:
 
-```
+```bash
+# 1. Clone this repository 
 git clone git@github.com:observablehq/plot.git
-cd plot
-yarn
+cd plot # cd into the folder
```

Comment:
Does the comment “cd into the folder” help here, or does it just repeat what the code is saying? Is the expectation that the reader doesn’t know what “cd” does, and if so, should the comment explain that in English? It’s probably more appropriate to assume the reader knows what “cd” does, along with other terminal basics, here.

Reply:
Good point, we can remove the annotation.